data/publiccloud/log_instance.sh

```diff
@@ -121,6 +121,22 @@ azure_stop_log()
     fi
 }
 
+openstack_start_log()
+{
+    read_unique_counter
+    set +e
+    openstack server start "$INSTANCE_ID" > "${OUTPUT_DIR}/$CNT""_boot_log_start.txt" 2>&1
+    set -e
+}
+
+openstack_stop_log()
+{
+    read_unique_counter
+    set +e
+    openstack server start "$INSTANCE_ID" > "${OUTPUT_DIR}/$CNT""_boot_log_start.txt" 2>&1
```

Comment:
was it meant to be `openstack server stop "$INSTANCE_ID" > "${OUTPUT_DIR}/$CNT""_boot_log_stop.txt" 2>&1`?

Reply:
Yup, as it is for others too

Reply:
just to clarify. i am not sure what do you mean /as it is for others too/. the command is exactly the same as in the `openstack_start_log`. in the `openstack_stop_log` you use `start` instead of `stop`, `pause` or whatever meant to be.

Reply:
ah, overlooked the server command. ctrl+c & crtl+v, thanks!